barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/goblin_translator_circuit_builder.cpp

```diff
@@ -28,11 +68,12 @@ GoblinTranslatorCircuitBuilder::AccumulationInput generate_witness_values(
      */
     auto base_element_to_bigfield = [](Fq& original) {
```

Comment:
I wonder if something like base_element_to_limbs would be more descriptive for people who aren't familiar with the details of bigfield